pkg/sriov/sriov_test.go

```diff
@@ -209,76 +209,6 @@ func (_m *MockNetlinkManager) LinkSetVfState(_a0 netlink.Link, _a1 int, _a2 uint
 	return r0
 }
 
-// mockPciUtils is an autogenerated mock type for the mockPciUtils type
```

Comment:
It seems we are not really using this mock in sriov tests, I presume it is safe to delete.
@martinkennelly @adrianchiris wdyt?

Reply:
I did not see `mockPciUtils` used.  Let me know if I missed something.  Happy to make changes.

Reply:
Sorry I did not realize that mockery created that file.  So I removed this change for now.

Reply:
I see its not being used so should be deleted.